source/operations/cert-manager/cert-manager-tenants.rst

```diff
@@ -0,0 +1,252 @@
+      spec:
+        ca:
+          secretName: tenant-1-ca-tls
+
+2. Apply the ``Issuer`` resource definition to the cluster.
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl apply -f tenant-1-ca-issuer.yaml
+
+3) Create a certificate for the tenant
+--------------------------------------
+
+Request that cert-manager issue a new TLS server certificate for MinIO.
+The certificate must be valid for the following DNS domains:
+
+- ``minio.<namespace>``
+- ``minio.<namespace>.svc``
+- ``minio.<namespace>.svc.<cluster domain>``
+- ``*.<tenant-name>-hl.<namespace>.svc.<cluster domain>``
+- ``*.<namespace>.svc.<cluster domain>``
+- ``*.<tenant-name>.minio.<namespace>.svc.<cluster domain>'``
+
+.. important::
+
+   Replace the the placeholder text (marked with the ``<`` and ``>`` characters) with values for your tenant: 
+
+   - ``<cluster domain>`` is the internal root DNS domain assigned in your Kubernetes cluster. 
+     Typically, this is ``cluster.local``, but confirm the value by checking your CoreDNS configuration for the correct value for your Kubernetes cluster. 
+      
+     For example:
+
+     .. code-block:: shell
+        :class: copyable
+
+        kubectl get configmap coredns -n kube-system -o jsonpath="{.data}"
+
+     Different Kubernetes providers manage the root domain differently.
+     Check with your Kubernetes provider for more information.
+
+   - ``tenant-name`` is the name provided to your tenant in the ``metadata.name`` of the Tenant YAML. 
+     For this example it is ``myminio``.
+
+   - ``namespace`` is the value created earlier where the tenant will be installed.
+     In the tenant YAML, it is defined in the the ``metadata.namespace`` field. 
+     For this example it is ``tenant-1``.
+
+1. Request a ``Certificate`` for the domains mentioned above
+
+   Create a file called ``tenant-1-minio-certificate.yaml`` with the following contents: 
+
+   .. code-block:: yaml
+      :class: copyable
+
+      # tenant-1-minio-certificate.yaml
+      apiVersion: cert-manager.io/v1
+      kind: Certificate
+      metadata:
+        name: tenant-certmanager-cert
+        namespace: tenant-1
+      spec:
+        dnsNames:
+          - "minio.tenant-1"
+          - "minio.tenant-1.svc"
+          - 'minio.tenant-1.svc.cluster.local'
+          - '*.minio.tenant-1.svc.cluster.local'
+          - '*.myminio-hl.tenant-1.svc.cluster.local'
+          - '*.myminio.minio.tenant-1.svc.cluster.local'
+        secretName: myminio-tls
+        issuerRef:
+          name: tenant-1-ca-issuer
+
+   .. tip::
+
+      For this example, the Tenant name is ``myminio``. 
+      We recommend naming the secret in the field ``spec.secretName`` as ``<tenant-name>-tls``, following the naming convention the MinIO Operator uses when creating certificates without cert-manager.
+
+2. Apply the certificate resource to the cluster.
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl apply -f tenant-1-minio-certificate.yaml
+
```

Comment:
They can validate by checking `kubectl get secrets` right? We might include that step

```
kubectl describe secret/myminio-tls -n tenant-1-whatever-namespace
```